plugin/trino-prometheus/src/test/java/io/trino/plugin/prometheus/TestPrometheusIntegration.java

```diff
@@ -119,13 +122,36 @@ public void testCorrectNumberOfSplitsCreated()
         config.setCacheDuration(new Duration(30, SECONDS));
         PrometheusTable table = client.getTable("default", "up");
         PrometheusSplitManager splitManager = new PrometheusSplitManager(client, new PrometheusClock(), config);
+        PrometheusSessionProperties sessionProperties = new PrometheusSessionProperties(config);
+        ConnectorSession session = TestingConnectorSession.builder()
+                .setPropertyMetadata(sessionProperties.getSessionProperties())
+                .build();
         ConnectorSplitSource splits = splitManager.getSplits(
                 null,
-                null,
+                session,
                 new PrometheusTableHandle("default", table.name()),
                 (DynamicFilter) null,
                 Constraint.alwaysTrue());
         int numSplits = splits.getNextBatch(NUMBER_MORE_THAN_EXPECTED_NUMBER_SPLITS).getNow(null).getSplits().size();
         assertThat((double) numSplits).isEqualTo(config.getMaxQueryRangeDuration().getValue(TimeUnit.SECONDS) / config.getQueryChunkSizeDuration().getValue(TimeUnit.SECONDS));
     }
+
+    @Test
+    public void testSessionProperties()
+    {
+        Session session = Session.builder(getSession())
+                .setCatalogSessionProperty("prometheus", "max_query_range_duration", "1d")
+                .setCatalogSessionProperty("prometheus", "query_chunk_size_duration", "1h")
+                .build();
+
+        assertQuery(session,
+                "show session like 'prometheus.max_query_range_duration'",
```

Comment:
We should test the session properties' behavior instead of listing values.

Reply:
In the same file, testCorrectNumberOfSplitsCreated is using SessionProperties and asserting on the correct number of splits.

I thought you wanted to make sure that we are able to set the session properties. That was the change made

Reply:
Hi @ebyhr , could you please have a look at my comment above. Thanks

Reply:
Hi @ebyhr , are you okay with already existing integration test using session properties testing the number of splits.

Reply:
It would be nice to rewrite the existing test or add a query-based test in the future. Could you add TODO comment in `testCorrectNumberOfSplitsCreated`?

This `testSessionProperties` isn't so helpful. You can remove this.